app/views/spaces/show.html.erb

```diff
@@ -16,7 +16,8 @@
 
 <%= link_to "VIEW ALL SPACES", spaces_path, class: 'btn btn-outline-secondary' %>
 <%= link_to 'DELETE THIS PLACE', space_path(@space), class: 'btn btn-outline-secondary', method: :delete, data: { confirm: "Are you sure you want to delete this space?" } %>
-<%= link_to "BOOK THIS SPACE", new_space_reservation_path(@space), class: 'btn btn-primary' %>
+<%= link_to "UPDATE/EDIT", edit_space_path(@space), class: 'btn btn-outline-secondary' %>
+<%= link_to "BOOK THIS SPACE", space_path(@space), class: 'btn btn-primary' %>
 
 <div class="map" style="background-image: url('https://maps.googleapis.com/maps/api/staticmap?center=<%= @space.location %>&zoom=13&scale=2&size=1000x1000&maptype=roadmap&markers=color:red%7C<%= @space.location %>&key=AIzaSyBtaHK-BLhBg7TruJu8iKk_wETqZMJM-GY')"></div>
```

Comment:
We should remove this line and replace it with mapbox at some stage (@CharlineNoga)